model_analyzer/config/generate/quick_run_config_generator.py

```diff
@@ -122,43 +146,80 @@ def set_last_results(self, measurements):
         """
 
         self._coordinate_data.increment_visit_count(self._coordinate_to_measure)
+        self._neighborhood.coordinate_data.increment_visit_count(
+            coordinate=self._coordinate_to_measure)
 
         if measurements is not None and measurements[0] is not None:
             assert len(measurements) == 1
 
             throughput = measurements[0].get_non_gpu_metric_value(
```

Comment:
move the debug printing into a helper function that is called before the if statement. The if/else itself should be tiny.

Reply:
for that matter, is the if/else even needed here once the debug code is extracted?

Reply:
The way the interface is designed, we may get a list of measurements. For example, in the brute force code we pick a model config, and then sweep over PA configs. That whole list of results is passed to the model config generator so it can make a decision

Reply:
I see. Thanks for the explanations. I have extracted the debug prints and removed the if/else statements (you're right, we don't need them anymore :))